test/OpenTelemetry.Instrumentation.Http.Tests/HttpClientTests.Basic.cs

```diff
@@ -425,6 +425,18 @@ public async Task HttpRequestMethodIsSetOnActivityAsPerSpec(string originalMetho
         }
 
         Assert.Equal(expectedMethod, activity.GetTagValue(SemanticConventions.AttributeHttpRequestMethod));
+
+#if NET9_0_OR_GREATER
```

Comment:
Can we make it runtime check?
In auto-instrumentation I would like to have only one copy of the library, for the lowest supported .NET version. It is valid also for other places.

Reply:
I think this is a common pattern in test projects. 
I don't think this would affect auto-instrumentation. Am I missing something?

Reply:
I missed that it is tests. Sorry :) . This comment should be valid only for production code.